### PR TITLE
Create attendee

### PR DESCRIPTION
**Pull request by agustinrhcp**
#176 #177 #178 #152

### CODE REVIEW

**Comment by rabble** [comment]
This seems to have broken viewing an person record. 

http://dev-advocacycommons-pr-191.herokuapp.com/members/2

File: app/models/person.rb [hunk 1]
```diff
@@ -37,6 +41,10 @@ def primary_email_address
     email_addresses.detect(&:primary?)&.address
   end
 
+  def primary_email_address=(email)
```
Comment (rabble) [inline app/models/person.rb]: How does this handle duplicates or finding if somebody already is in our records?

File: client/app/bundles/Events/actions/MembersActions.jsx [hunk 2]
```diff
@@ -22,3 +23,12 @@ export const fetchMember = (id) => {
     payload: request
   };
 };
+
```
Comment (rabble) [inline client/app/bundles/Events/actions/MembersActions.jsx]: When tested here: http://dev-advocacycommons-pr-191.herokuapp.com/events/1/attendances/new#

This isn't working.

File: client/app/bundles/Events/reducers/AttendanceFormReducer.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  'family-name': '',
+  'primary-phone-number': '',
+  'disabled': ''
+};
+
+export default (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+  case SET_ATTENDANCE_ATTRIBUTE:
+    const { prop, value } = action.payload;
+    return { ...state, [prop]: value };
```
Comment (rabble) [inline client/app/bundles/Events/reducers/AttendanceFormReducer.jsx]: { ...state, [prop]: value };
 Now that's a weird data structure which i haven't seen before.

**Comment by agustinrhcp** [comment]
I already fixed the person record view. I'll push it on Monday, I messed with all those conflicts.

File: client/app/bundles/Events/reducers/AttendanceFormReducer.jsx (hunk 3 above)
Comment (agustinrhcp) [inline client/app/bundles/Events/reducers/AttendanceFormReducer.jsx]: ES6 syntax. 
if props = 'email' and value = 'test@test.com' you'll get everything that's on state + email: 'test@test.com'

File: app/models/person.rb (hunk 1 above)
Comment (agustinrhcp) [inline app/models/person.rb]: It doesn't yet, I'm just avoiding duplication in the controller for now

**Comment by agustinrhcp** [comment]
This should be ready for testing

**Pull request merged**